dashboard/src/components/DeploymentForm/DeploymentBinding.tsx

```diff
@@ -82,7 +48,8 @@ class DeploymentBinding extends React.Component<IDeploymentBindingProps, IDeploy
   public onBindingChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
     this.setState({
       selectedBinding:
-        this.props.bindings.find(binding => binding.metadata.name === e.target.value) || undefined,
+        this.props.bindingsWithSecrets.find(b => b.binding.metadata.name === e.target.value) ||
```

Comment:
`find` already returns `undefined` if not found

Reply:
Didn't want to change the original code much, but I can remove that unnecessary `|| undefined`